editor/configs/fatalitiesListViewColumns.tsx

```diff
@@ -75,4 +75,16 @@ export const fatalitiesListViewColumns: ColDataCardDef<fatalitiesListRow>[] = [
     label: "Engineering Area",
     sortable: true,
   },
+  {
+    path: "latitude",
+    label: "latitude",
+    exportOnly: true,
+    fetchAlways: true,
+  },
+  {
+    path: "longitude",
+    label: "longitude",
+    exportOnly: true,
+    fetchAlways: true,
```

Comment:
im not sure if this is needed, but i added this fetchAlways. @johnclary can u explain this prop to me?

Reply:
ooo thank u for explaining chia :pray:  so i do indeed need it here